uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -64,13 +65,17 @@ define([
         }
       };
       vm.uiOptions = {
-        isAdmin: false,
         isChangeExpanded: false,
+        isManager: false, //when the dialog is opened by manager or admin
+        isOwner: false, //when the dialog is opened by the owner
         multipleDays: true,
+        contact: '',
         showDatePickerFrom: false,
         showDatePickerTo: false,
         userDateFormat: HR_settings.DATE_FORMAT,
+        selectedStatus: undefined,
         showBalance: false,
+        statusLabel: '',
```

Comment:
its a one time call needed only in init like this `vm.uiOptions.statusLabel = getStatusFromValue(vm.leaveRequest.status_id).label;`

getStatusFromValue is a private function so either I will have to make that public or keep a public attribute, my preference would be for a simple attribute on UI rather than complex code there